suites/fs/snaps/tasks/snaptests.yaml

```diff
@@ -2,6 +2,20 @@ tasks:
 - workunit:
     clients:
       all:
-        - snaps/snaptest-0.sh
-        - snaps/snaptest-1.sh
-        - snaps/snaptest-2.sh
+        - fs/snaps/snaptest-0.sh
+        - fs/snaps/snaptest-1.sh
+        - fs/snaps/snaptest-2.sh
+        - fs/snaps/snap-rm-diff.sh
+        - fs/snaps/snaptest-authwb.sh
+        - fs/snaps/snaptest-capwb.sh
+        - fs/snaps/snaptest-dir-rename.sh
+        - fs/snaps/snaptest-double-null.sh
+        - fs/snaps/snaptest-estale.sh
+        - fs/snaps/snaptest-git-ceph.sh
+        - fs/snaps/snaptest-intodir.sh
+        - fs/snaps/snaptest-multiple-capsnaps.sh
+        - fs/snaps/snaptest-snap-rename.sh
+        #- fs/snaps/snaptest-snap-rm-cmp.sh
```

Comment:
Is there a tracker ticket for the reason this is commented out?  Ideally would mention in in a comment in this file